src/main/scala/PcapExample.scala

```diff
@@ -2,13 +2,7 @@
 package com.scalawilliam.scalanative
 
 import scala.scalanative.native
-import scala.scalanative.native.{
-  CString,
-  CUnsignedInt,
-  Ptr,
-  fromCString,
-  toCString
-}
+import scala.scalanative.native._
```

Comment:
Article will need to be 'reloaded' if line numbers change in the source code.
Eg look at:
https://github.com/ScalaWilliam/ScalaWilliam.com/blob/ded2a73f9591d549aecc8dc31fbcf54804ce4559/scala-native-libpcap/index.html#L252

```html
<code data-source="src/main/scala/pcap.scala" data-from="9" data-to="36">@native.link("pcap")
...
```
Cheap way to embed code.

If you take a look at https://github.com/ScalaWilliam/ScalaWilliam.com/blob/master/scala-native-libpcap/source.py you'll see what I mean.

So you'll need to run `SOURCE_ROOT=/path/to/scala-native-libpcap-root/ python source.py` and it'll update `index.html`.

Unfortunately I didn't document this as wasn't planning for the need. There are two options:
- I document it for you
- You create a ticket, I pay $5 for it & another $5 to complete it.

It'd involve creating a python requirements.txt (for the html manipulation lib I'm using) and README.md with instructions (git clone, pip install, python source.py).